empathyco/x
# feat: filters in history queries

# Pull request template
 
We want to save filters in history queries. At the moment we are only saving the query and total results, so when you search for a query and apply filters to it, it means that these filters are important in your search. So, why don't we save them? We think that it would be a useful feature for the user. 

The idea is to search for a query, apply filters and remove it. Then, if you click on the history query, the search request should contain the query and the filters applied previously.

## Motivation and context




- [X] Open issue. If applicable, link: [EMP-2208](https://searchbroker.atlassian.net/browse/EMP-2208)

## Type of change


## What is the destination branch of this PR?


## How has this been tested?


You can follow these steps:
- Search for a query
- Apply filters
- Remove query
- Select the same query in the history queries column
- Check if the filters are applied 

## Checklist:



[EMP-2208]: https://searchbroker.atlassian.net/browse/EMP-2208?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
